activerecord/lib/active_record/connection_adapters/abstract/query_cache.rb

```diff
@@ -134,7 +134,6 @@ def cache_notification_info(sql, name, binds)
             binds: binds,
             type_casted_binds: -> { type_casted_binds(binds) },
             name: name,
-            connection_id: object_id,
```

Comment:
Should we remove the same info in `AbstractAdapter#log` too, and add the `connection: self` instead?

https://github.com/rails/rails/blob/0542e0608f86d9e4089861f4e72c578bc983f89f/activerecord/lib/active_record/connection_adapters/abstract_adapter.rb#L688-L689

Reply:
We should.  I'm not sure why we pass the object id though.  If it's for an identifier, maybe we should add a UUID to the connection?

Reply:
It was to make possible to the subscriber to find the connection by the if. Since we are passing the connection we don't need to pass anymore.